hofx/stage_obs.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+from solo.logger import Logger
+from solo.basic_files import mkdir
+from r2d2 import fetch
+from genyaml import gen_yaml as get_config
+import click
+
+logger = Logger('stageObs')
+@click.command()
+@click.argument('expdir', required=True)
+def stage_obs(expdir):
+    # stage observations based on environment vars
+    # and vars set in config YAMLs in experiment dir
+    # get config
+    config = get_config('stageObs', expdir, quiet=True)
+    logger.info(f"Preparing to stage observations for {config['window begin']}")
+    mkdir(config['obs_dir'])
+    for ob in config['observations']:
+        obname = ob['obs space']['name'].lower()
+        outfile = ob['obs space']['obsdatain']['obsfile']
+        # try to grab obs
+        fetch(
+            type='ob',
+            provider=config['obs_src'],
+            experiment=config['obs_dump'],
+            date=config['window begin'],
+            obs_type=obname,
+            time_window=config['window length'],
+            target_file=outfile,
+            ignore_missing=True,
+            database=config['obs_db'],
```

Comment:
for aid in debugging, add:
```
full_report  = yes,
report = f"fetch_obs_{config['window begin']}.yaml"
```
`config['window begin'] = date`

Reply:
One for each obs type (it runs in a loop currently)?

Reply:
I don't think you need to loop over obs. r2d2 can fetch in parallel.  I missed the loop over obs.

Reply:
full_report and report didn't work in stage_obs or stage_bkg... Getting the same schema issues like before.

The loop was the simplest way to do it since the YAML is a list which then has obs bias, obsdatain, etc. defined in it. Not using a list will require more scripting at this level but yes I know R2D2 can grab 'all' obs.

Reply:
Ok. let me take a look.  It should work.

Reply:
@aerorahul for the sake of the 'tutorial' this afternoon, is it fine if I merge this in with the shebangs removed and keep the full_report parts commented out for now?